code/modules/core_implant/cruciform/rituals/priest.dm

```diff
@@ -239,3 +239,97 @@
 
+
+///////////////////////////////////////
+///////////SHORT BOOST LITANIES////////
+///////////////////////////////////////
+
+/datum/ritual/cruciform/priest/short_boost
+	name = "Short boost ritual"
+	phrase = null
+	desc = "This litany boosts mechanical stats of everyone who's hear you on the short time. "
+	cooldown = TRUE
+	cooldown_time = 2 MINUTES
+	effect_time = 2 MINUTES
+	cooldown_category = "short_boost"
+	var/list/stats_to_boost = list()
+
+	New()
+		..()
+		desc = "This litany boosts [get_stats_to_text()] stats of everyone who's hear you on the short time."
+
+
+/datum/ritual/cruciform/priest/short_boost/pre_check(mob/living/carbon/human/H, obj/item/weapon/implant/core_implant/C, targets)
+	if(cooldown && is_on_cooldown(H))
+		fail("Litanies of this type can't be spoken too often.", H, C)
+		return FALSE
+	return TRUE
+
+
+/datum/ritual/cruciform/priest/short_boost/perform(mob/living/carbon/human/user, obj/item/weapon/implant/core_implant/C)
+	var/list/people_around = list()
+	for(var/mob/living/carbon/human/H in view(user))
+		if(H != user && !isdeaf(H))
+			people_around.Add(H)
+
+	if(people_around.len > 0)
+		user << SPAN_NOTICE("Your cruciform emmits a special signal.")
+		playsound(user.loc, 'sound/machines/signal.ogg', 50, 1)
+		for(var/mob/living/carbon/human/participant in people_around)
+			participant << SPAN_NOTICE("You hear a silent signal...")
+			give_boost(participant)
+		set_global_cooldown()
+		return TRUE
+	else
+		fail("Not enough hearers.", user, C)
+		return FALSE
+
+
+/datum/ritual/cruciform/priest/short_boost/proc/give_boost(mob/living/carbon/human/participant)
+	for(var/stat in stats_to_boost)
+		var/amount = stats_to_boost[stat]
+		participant.stats.changeStat(stat, amount)
+		addtimer(CALLBACK(src, .proc/take_boost, participant, stat, amount), effect_time)
+	spawn(30)
+		participant << SPAN_NOTICE("You feel a little dizziness at the moment, but later you got an extraordinary understanding of [get_stats_to_text()].")
+
+
+/datum/ritual/cruciform/priest/short_boost/proc/take_boost(mob/living/carbon/human/participant, stat, amount)
+	participant.stats.changeStat(stat, -amount)
+	participant << SPAN_WARNING("You almost forgot something that you were got of [get_stats_to_text()]. The last piece of that knoweledge has disappeared.")
+
+/datum/ritual/cruciform/priest/short_boost/proc/get_stats_to_text()
+	if(stats_to_boost.len == 1)
+		return lowertext(stats_to_boost[1])
+	var/stats_text = ""
+	for(var/i = 1 to stats_to_boost.len)
+		var/stat = stats_to_boost[i]
+		if(i == stats_to_boost.len)
+			stats_text += " and [stat]"
+			continue
+		if(i == 1)
+			stats_text += "[stat]"
+		else
+			stats_text += ", [stat]"
+	return lowertext(stats_text)
+
+
+/datum/ritual/cruciform/priest/short_boost/mechanical
+	name = "Pounding Whisper"
+	phrase = "Vocavitque nomen eius Noe dicens iste consolabitur nos ab operibus et laboribus manuum nostrarum in terra cui maledixit Dominus"
+	stats_to_boost = list(STAT_MEC = 4)
```

Comment:
4 is very little, especially for such a short duration
Try a value of 20 for all of these

Reply:
I think 20 will be too much. Let's stay on 10.